ui/src/dashboards/components/GraphOptionsDecimalPlaces.tsx

```diff
@@ -42,6 +42,7 @@ class GraphOptionsDecimalPlaces extends PureComponent<Props> {
         <label> Decimal Places </label>
         <OptIn
           min="0"
+          max="20"
```

Comment:
Would you consider making this a constant in the constants file?

Reply:
will do thanks!